src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -0,0 +1,120 @@
+<%--
+  Copyright 2018 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+<%@ page import="java.time.Instant" %>
+<%@ page import="java.util.Date" %>
+<%@ page import="java.util.Calendar" %>
+<%@ page import="java.util.TimeZone" %>
+<%@ page import="java.text.DateFormat" %>
+<%@ page import="java.text.SimpleDateFormat" %>
+<%@ page import="java.util.List" %>
+<%@ page import="java.util.Map" %>
+<%@ page import="java.util.UUID" %>
+<%@ page import="codeu.controller.ActivityFeedServlet" %>
+<%@ page import="java.util.PriorityQueue" %>
+<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="codeu.model.data.Message" %>
+<%@ page import="codeu.model.data.User" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+<%@ page import="codeu.model.store.basic.ConversationStore" %>
+<%
+PriorityQueue<ActivityFeedServlet.Item> FeedItem = (PriorityQueue<ActivityFeedServlet.Item>) request.getAttribute("FeedItems");
+Map<UUID, Object> ids = (Map<UUID, Object>) request.getAttribute("ids");
+//List<User> users = (List<User>) request.getAttribute("users");
+//List<Conversation> conversations = (List<Conversation>) request.getAttribute("conversations");
+//List<Message> messages = (List<Message>) request.getAttribute("messages");
+%>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CodeU Chat App</title>
+  <link rel="stylesheet" href="/css/main.css">
+
+  <style>
+    #activity {
+        background-color: white;
+        height: 500px;
+        overflow-y: scroll
+    }
+  </style>
+
+  <script>
+    // scroll the activity feed div to the bottom
+    function scrollChat() {
+       var chatDiv = document.getElementById('activity');
+       chatDiv.scrollTop = chatDiv.scrollHeight;
+    };
+  </script>
+</head>
+<body onload="scrollChat()">
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+  </nav>
+
+  <div id="container">
+
+    <h1><%= "Activity Feed" %>
+      <a href="" style="float: right">&#8635;</a></h1>
```

Comment:
what's the anchor for?

Reply:
I didn't need it at the end, deleted. Done